src/control/common/proto/ctl/firmware.pb.go

```diff
@@ -6,8 +6,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.6.1
+// 	protoc-gen-go v1.28.1
+// 	protoc        v3.5.0
```

Comment:
These version-only changes to generated files should not be part of the commit. You can revert them locally by doing `git checkout master -- path/to/pb.go` for the files that only have version updates, then pushing that commit so that they are not included in the landing commit.